tools/cli/commands/build.js

```diff
@@ -919,10 +920,21 @@ async function buildProject( t ) {
 		crlfDelay: Infinity,
 	} );
 
-	rl.on( 'line', line => {
+	rl.on( 'line', async line => {
 		const match = line.match( /^## +(\[?[^\] ]+\]?)/ );
 		if ( match && match[ 1 ] ) {
-			projectVersionNumber = match[ 1 ].replace( /[[\]]/g, '' );
+			projectRunVersionNumber = projectVersionNumber = match[ 1 ].replace( /[[\]]/g, '' );
+			if ( t.project.startsWith( 'packages/' ) && projectVersionNumber.endsWith( 'alpha' ) ) {
+				const ts = (
+					await t.execa( 'git', [ 'log', '-1', '--format=%ct', '.' ], {
+						cwd: t.cwd,
+						stdio: [ null, 'pipe', null ],
+					} )
+				).stdout;
```

Comment:
Apparently prettier is being stupid. Sigh.
```suggestion
				 ).stdout;
```